bots/image-refresh

```diff
@@ -57,6 +57,8 @@ TRIGGERS = {
         "fedora-30/firefox@weldr/cockpit-composer",
         "fedora-30/edge@weldr/cockpit-composer",
     ],
+    "fedora-31": [
```

Comment:
This commit should be squashed into the main one. We need to separately declare in `testmap.py` and land that before we can test this PR.